FCM single/multiple notification 기능 생성

cafitac opened the pull request:
push/fcm_notify_api/notification.py
 - ADD: Notify class 생성
 - ADD: staticmethod (single_notify, multiple_notify) 생성
 - ADD: 실제로 notification 로직을 수행하는 (notify_single_device, notify_multiple_devices) method 생성

hanseongd commented on scheduler/push/fcm_notify_api/notification.py:
```diff
@@ -0,0 +1,28 @@
+    def notify_multiple_devices(self):
+        self.push_service.notify_multiple_devices(registration_ids=self.registration_ids, message_title=self.message_title, message_body=self.message_body, data_message=self.data_message)
+
+    @staticmethod
```
여기서 single_notify, multiple_notify를 만들게된 이유, 꼭 필요한지 를 모르겠어

cafitac replied:
그냥 단순하게 pyfcm에서 single_notify랑 multiple_notify로 나눠놔서 그거 따라가려고 저렇게 구조를 짰던 것 같아요...
지금 다시 생각해보니 한 사람이 device를 두 개 이상 가지고 있을 수도 있다는 생각이 들어서 notify_multiple_devices에 하나의 device_id만 넘겨서 줘도 알림이 가는 걸 확인. 그냥 모두 multiple_notify로 통일하기로 했습니당

Commit:
User (Email, Nickname) validate 뷰에서 request body 받는 부분, response 수정

member/views.py
 - MOD: request.POST.get('email') -> request.data['email']
 - MOD: request.POST.get('nickname') -> request.data['nickname']
 - MOD: (Email, Nickname) 닉네임 사용 여부의 response status code가 반대로 되어있어 변경
